.travis.yml

```diff
@@ -57,10 +55,7 @@ matrix:
     env: TOXENV='isort'
   - python: 3.6
     env: TOXENV='docs'
-
-  - python: 2.7
-    env: DJANGO='django22' CMS='cms37'
   - python: 2.7
-    env: DJANGO='django21' CMS='cms37'
+    env: DJANGO='django22' CMS='cms40' FE=1
```

Comment:
Django 2+ is not Python 2.7 compatible. This is wrong in the base project.

"Django 2.0 supports Python 3.4, 3.5, 3.6, and 3.7. We highly recommend and only officially support the latest release of each series."
https://docs.djangoproject.com/en/2.0/releases/2.0/

Reply:
Delete this

Reply:
if you see this.. this is basically excluding this test..
It's under "exclude:" matrix

Reply:
Agreed, my bad. Retract my comment.